### PR TITLE
[minor] Fixing some warnings

### PR DESCRIPTION
**Pull request by dhavalshreyas**
### What did you change and why?
Fixing some Xcode warnings.

### Potential risks introduced?
None

### What tests were performed (include steps)?
Compile and tests.

### Checklist

- [ ] Unit/UI tests have been written (if necessary)
- [x] Manually tested

### CODE REVIEW

File: Sources/NTPConnection.swift [hunk 1]
```diff
@@ -243,7 +243,7 @@ private extension NTPConnection {
                 return
             }
 
-            let packet = data.withUnsafeBytes { $0.pointee as ntp_packet_t }.nativeEndian
+            let packet = data.withUnsafeBytes { $0.load(as: ntp_packet_t.self) }.nativeEndian
```
Comment (rvanmelle) [inline Sources/NTPConnection.swift]: have never used `load(as:` ... so ... sure!

**Comment by Ariandr** [comment]
@dhavalshreyas @rvanmelle 
Hello guys, could you please update the pod to include these changes? Even the version 5.0.1 hasn't been released as a pod yet.
Thanks.

**Comment by 00FA9A** [comment]
@Ariandr as explained in https://github.com/instacart/TrueTime.swift/issues/58, we are in the process of transferring pod ownership. Once that is completed, pod will be updated.

**Comment by Ariandr** [comment]
@00FA9A 
Okay, thank you!

**Pull request merged**